src/utils/data.ts

```diff
@@ -0,0 +1,50 @@
+export type QuestionData = QuestionDataSingle[];
```

Comment:
tu spoko, ale ja osobiście nie jestem fanem przypisywania typu tablicowego do jakiejś innej nazwy. Lepiej po prostu używać QuestionDataSingle[] tam gdzie to konieczne
Wtedy możesz dać nazwę QuestionData po prostu